hydpy/docs/draw_model_sketches.py

```diff
@@ -420,11 +420,11 @@ def plot(self, frame: Frame) -> None:
 
 
 class Soil(Element):
-    version: Literal["v1_v2", "v3_v4"] = "v1_v2"
+    version: Literal["v1_base", "v3_v4"] = "v1_base"
```

Comment:
What do you mean by "v1_base"?  I would expect just "v1" if functionality is only relevant to `hland_v1` or maybe "base" if relevant to all `hland` application models.

Reply:
This logic was unfortunately not consistently implemented before, as for example in line 1234 elements_hland_v3 were derived from elements_hland_v1_v2.

Reply:
Okay, it's my fault, then.  Maybe it's okay to leave it as is.  `draw_model_sketches` needs major refactoring or maybe replacement by a less custom approach anyhow.  But that has low priority.